andela-stuff/andela-teams-2
# #153893988 Setup testing for the client side

#### What does this PR do?
Sets up testing for the front-end

#### Description of the task to be completed?
- install enzyme
- install jest
- configure enzyme to work with jest
- write base test to ensure everything is working as expected

#### How should this be manually tested?
After cloning the application, run the command `npm test`
- This should run the only single test in the application.

#### Screenshots, if any
<img width="1133" alt="screen shot 2018-08-13 at 6 35 19 pm" src="https://user-images.githubusercontent.com/24798364/44047876-ad838fc0-9f27-11e8-96d5-23fe20fe31e7.png">
